css/style.css

```diff
@@ -0,0 +1,128 @@
+html,
+body {
+  min-width: 290px;
+  color: #4d4e53;
+  background-color: #fff;
```

Comment:
Background color was #ffffff; and now it is changed to a colled color code #fff;

Reply:
please it is better to always start with courteous words like please or kindly. know that it is not an order of that you know rather it is an orientation.